tools/netusb.sh

```diff
@@ -0,0 +1,61 @@
+#!/bin/bash
+set -x
+
+# USB Ethernet Gadget interface
```

Comment:
Add some description in tools/README.txt

Reply:
@xiaoxiang781216 will fix